<launch>
  <node pkg="perception_node" type="lane_center_node" name="lane_center_node" output="screen"/>
  <node pkg="perception_node" type="lane_color_node" name="lane_color_node" output="screen"/>

  <!-- new uupdate _ gate -->
  <node pkg="perception_node" type="gate_camera_node" name="gate_camera_node" output="screen"/>
  <node pkg="perception_node" type="crosswalk_node" name="crosswalk_node" output="screen"/> 
</launch>
